source/indexes.txt

```diff
@@ -81,8 +81,6 @@ Create an Index
 
 .. include:: /includes/shared/steps/create-index.rst
 
-.. BEGIN-COMPASS-ONLY
-
 Create an Atlas Search Index
```

Comment:
Per Betsy's request, we're keeping this information here, so users on Data Explorer are directed to the correct place to create Search/Vector Search indexes.

Reply:
Create a Search Index Page for Atlas: https://www.mongodb.com/docs/atlas/atlas-search/tutorial/create-index/

Reply:
Note that these links will be swapped out in https://github.com/10gen/cloud-docs/pull/6554